tasks/configure.yml

```diff
@@ -1,10 +1,11 @@
 ---
 
-- name: Set hostname
+- name: Ensure hostname is resolvable
   become: yes
   lineinfile:
     dest: /etc/hosts
-    line: "127.0.0.1 {{ ansible_hostname }}"
+    line: "127.0.0.1 localhost {{ ansible_hostname }}"
```

Comment:
This role is not responsible for setting hostname. This tasks only ensure that Go CD can resolve hostname to IP address. Nothing else, nothing more.